src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/MediaElement.shared.cs

```diff
@@ -179,6 +187,11 @@ int IMediaElementController.VideoWidth
 			get => (double)GetValue(VolumeProperty);
 			set => SetValue(VolumeProperty, value);
 		}
+		double IMediaElementController.Speed
+		{
+			get => (double)GetValue(SpeedProperty);
+			set => SetValue(SpeedProperty, value);
+		}
```

Comment:
What's the purpose of adding this if this property is implemented implicitly? (And does the same)

Reply:
Removed that too 👍